stable/falco/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: falco
-version: 0.7.10
+version: 0.8.0
```

Comment:
Why 0.8.0?

Reply:
It's a new feature, minor bump makes sense to me